var/spack/repos/builtin/packages/minimap2/package.py

```diff
@@ -40,4 +48,6 @@ def install_minimap2(self):
         make(*make_arg)
         mkdirp(prefix.bin)
         install("minimap2", prefix.bin)
+        if self.spec.satisfies("js_engine=node"):
+            filter_file(r"k8", "node", "./misc/*.js")
```

Comment:
Do globa characters work in `filter_file`? Might be worth double checking that the conversion actually happens.